.github/workflows/update_pytorch_labels.yml

```diff
@@ -14,7 +14,10 @@ jobs:
     if: ${{ github.repository == 'pytorch/pytorch' }}
     steps:
       - name: Checkout PyTorch
-        uses: zhouzhuojie/checkout@05b13c9a0d21f08f6d5e64a1d5042246d13619d9
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
```

Comment:
why not `malfet/checkout`?

Reply:
Not sure, I guess whenever we can use action, we probably should, but let me change for the sake of uniformity.